docs/blog/newsletter-june-2018.rst

```diff
@@ -0,0 +1,122 @@
+* Windows Dev Center's `Messages <https://msdn.microsoft.com/en-us/library/windows/desktop/dn742469(v=vs.85).aspx>`_
+
+*For other style or mechanics questions, the* `#general <https://writethedocs.slack.com/messages/general>`_ *channel is full of helpful documentarians ready with guidance.*
+
+-----------------------
+Job Board: Your ad here
+-----------------------
+
+Although the newsletter is taking a summer break, the `Write the Docs job board <https://jobs.writethedocs.org/>`_ isn't going anywhere! Create an employer profile and post your open positions to get them in front of thousands of docs folk all around the world. Any jobs posted as a 'featured' listing will be included in next newsletter, in early August.
+
+-------------------------
+Upcoming community events
+-------------------------
+
+June 12 – Boston, MA, USA – `Joint Networking Event with Women Who Code <https://www.meetup.com/Write-the-Docs-BOS/events/251281050/>`_
+
+June 12 – Portland, OR, USA – `June Post-Conference Meetup <https://www.meetup.com/Write-The-Docs-PDX/events/250814073/>`_
+
+June 14 – Salt Lake City, UT, USA – `Writing the Docs for Multiple Projects <https://www.meetup.com/Write-the-Docs-SLC/events/251266405/>`_
+
+June 19 – Austin, TX, USA – `Panel Discussion: Navigating Career Paths <https://www.meetup.com/WriteTheDocs-ATX-Meetup/events/249817330/>`_
+
+June 21 – Boise, ID, USA – `Write the Docs Conference Rundown <https://www.meetup.com/Write-the-Docs-Boise/events/249633985/>`_
+
+June 26 – Ottawa, ON, Canada – `Auditing documentation | Developing Shopify's API docs style guide <https://www.meetup.com/Write-The-Docs-YOW-Ottawa/events/250880001/>`_
+
+June 26 – San Francisco, CA, USA – `The Developer Experience for APIs <https://www.meetup.com/Write-the-Docs-SF/events/251444527/>`_
+
+June 27 – Brisbane, Australia – `Facebook, Dynamite, Uber, Bombs, and Tech Writers: A quick historical excursion through ethics and documentation <https://www.meetup.com/Write-the-Docs-Australia/events/250630480/>`_
+
+June 27 – Manchester, UK – `WTD North England Social Event <https://www.meetup.com/Write-the-Docs-North/events/251039358/>`_
+
+June 28 – Denver, CO – `Docs and Drinks <https://www.meetup.com/Write-the-Docs-Boulder-Denver/events/251107194/>`_
+
+July 3 – Sydney, Australia – `Content strategy and doc fixits <https://www.meetup.com/Write-the-Docs-Australia/events/250791637/>`_
+
+July 4 – Melbourne, Australia – `Document the API | Technical Writing vs Knowledge Management <https://www.meetup.com/Write-the-Docs-Australia/events/250605610/>`_
+
+July 9 – Karlsruhe, Germany – `Code comments. Yay or nay? <https://www.meetup.com/Write-the-Docs-Karlsruhe/events/250444386/>`_
+
+July 31 – Ottawa, ON, Canada – `July Meetup <https://www.meetup.com/Write-The-Docs-YOW-Ottawa/events/251110543/>`_
```

Comment:
Just a note that they July 31 Ottawa meet up has the exact same info as their June meet up. I don't know if this is a Meetup.com error maybe?

Reply:
I did notice that, and I think it's just cuz it's a recurring meetup (fourth Wednesday or whatever of every month). I figured I'd include it, and assume they'll probably update the info before the event.